src/models/ITestType.ts

```diff
@@ -1,103 +1,62 @@
-    forVehicleSize: ForVehicleSize[] | null;
-    forVehicleConfiguration: ForVehicleConfiguration[] | null;
-    forVehicleAxles: number[] | null;
-    forEuVehicleCategory: ForEuVehicleCategory[] | null;
-    forVehicleClass: string[] | null;
-    forVehicleSubclass: ForVehicleSubclass[] | null;
-    forVehicleWheels: number[] | null;
-    testTypeClassification?: TestTypeClassification;
-    testCodes?: TestCode[];
-    nextTestTypesOrCategories?: NextTestTypesOrCategory[];
-}
-
-export enum ForVehicleConfiguration {
-    Articulated = "articulated",
-    Rigid = "rigid",
-    CentreAxleDrawbar = "centre axle drawbar",
-    SemiCarTransporter = "semi-car transporter",
-    SemiTrailer = "semi-trailer",
-    LowLoader = "low loader",
-    Other = "other",
-    Drawbar = "drawbar",
-    FourInLine = "four-in-line",
-    Dolly = "dolly",
-    FullDrawbar = "full drawbar"
-}
-
-export enum ForVehicleSize {
-    Small = "small",
-    Large = "large",
-}
-
-export enum ForVehicleType {
-    Hgv = "hgv",
-    Psv = "psv",
-    Trl = "trl",
-    Car = "car",
-    Lgv = "lgv",
-    Motorcycle = "motorcycle"
-}
+import {
+  FOR_VEHICLE_TYPE,
+  FOR_VEHICLE_SIZE,
+  FOR_VEHICLE_CONFIGURATION,
+  FOR_EU_VEHICLE_CATEGORY,
+  FOR_VEHICLE_SUB_CLASS,
+  TEST_TYPE_CLASSIFICATION
+} from "../assets/Enums";
 
-export enum ForEuVehicleCategory {
-    N2 = "n2",
-    N3 = "n3",
-    O1 = "o1",
-    O2 = "o2",
-    O3 = "o3",
-    O4 = "o4"
-}
-
-export enum ForVehicleSubclass {
-    A = "a",
-    C = "c",
-    S = "s",
-    L = "l",
-    M = "m",
-    N = "n",
-    P = "p",
-    T = "t",
-    R = "r"
-}
-
-export interface NextTestTypesOrCategory {
-    id: string;
-    linkedIds: string[] | null;
-    name: string;
-    testTypeName?: string;
-    forVehicleType: ForVehicleType | ForVehicleType[];
-    forVehicleSize: ForVehicleSize[] | null;
-    forVehicleConfiguration: ForVehicleConfiguration[] | null;
-    forVehicleAxles: number[] | null;
-    forEuVehicleCategory: ForEuVehicleCategory[] | null;
-    forVehicleClass: string[] | null;
-    forVehicleSubclass: ForVehicleSubclass[] | null;
-    forVehicleWheels: number[] | null;
-    testTypeClassification?: TestTypeClassification;
-    testCodes?: TestCode[];
-    nextTestTypesOrCategories?: NextTestTypesOrCategory[];
-}
+export interface ITestType {
+  id: string;
+  linkedIds: string[] | null;
+  name: string;
+  testTypeName?: string;
+  forVehicleType: FOR_VEHICLE_TYPE[];
+  forVtmOnly: boolean;
+  forVehicleSize: FOR_VEHICLE_SIZE[] | null;
+  forVehicleConfiguration: FOR_VEHICLE_CONFIGURATION[] | null;
+  forVehicleAxles: number[] | null;
+  forEuVehicleCategory: FOR_EU_VEHICLE_CATEGORY[] | null;
+  forVehicleClass: string[] | null;
+  forVehicleSubclass: FOR_VEHICLE_SUB_CLASS[] | null;
+  forVehicleWheels: number[] | null;
+  testTypeClassification?: TEST_TYPE_CLASSIFICATION;
+  testCodes?: TestCode[];
+  nextTestTypesOrCategories?: ITestType[];
+}
+
+// export interface NextTestTypesOrCategory {
```

Comment:
Do we need this commented out?
We can remove it if unused please

Reply:
removed